src/store/vaults.ts

```diff
@@ -56,6 +56,9 @@ export type VaultKey = string;
 export const keyForVault = (managerId: string, vaultId: string) =>
   `${managerId}.${vaultId}` as VaultKey;
 
+// XXX: find a way to get this from zoe.
```

Comment:
This is good for now. https://github.com/Agoric/agoric-sdk/pull/6884 will provide it. Consider an `UNTIL` comment.

Reply:
Done